src/steamship/cli/cli.py

```diff
@@ -402,6 +412,43 @@ def logs(
     )
 
 
+@click.command(name="prod")
```

Comment:
nit: i think we should use a different name. `prod` implies production, which is not exactly aligned with this use case (as I understand it). i get that you want to distinguish it from `local`, but I think this will lead to confusion (staging envs, etc.).

wdyt about `ship launch` ?

or even: `ship run local` and `ship run remote` ?

Reply:
I really like `run local` and `run remote`, but my worry is the reverse -- that `run remote` actually requires a full deployment (though the API technically supports a deployment that wouldn't become the default for new instances).

I think my dream solution would be something like:

run/serve local
run/serve remote <- deploys without making the version the default for new instances, then creates an instnace publish <- publishes the new version

I'm pretty OK with this being an evolving thing. What it is that they say? The hardest part of computer science is naming things?